services/registration/src/routes/index.ts

```diff
@@ -1,7 +1,9 @@
 import express from "express";
 
+import { applicationRouter } from "./application";
 import { branchRouter } from "./branch";
 
 export const defaultRouter = express.Router();
 
-defaultRouter.use("/", branchRouter);
+defaultRouter.use("/branch", branchRouter);
+defaultRouter.use("/application", applicationRouter);
```

Comment:
Can we make these route names plural? That's typically how the HTTP REST specification is, so like `/branches` and `/applications`